delivery/http/routing/userrouter.go

```diff
@@ -3,18 +3,19 @@ package routing
 import (
 	"encoding/json"
 	"errors"
+	"net/http"
+
 	"github.com/go-playground/validator/v10"
 	"github.com/gorilla/context"
 	"github.com/gorilla/mux"
-	"kanca-studio/palang/delivery/http/schema"
-	"kanca-studio/palang/manager"
-	"net/http"
+	"github.com/kanca-studio/palang/delivery/http/schema"
+	"github.com/kanca-studio/palang/manager"
 )
 
 var validate *validator.Validate
-var userManager manager.UserManager
+var userManager manager.User
 
-func UserRouter(router *mux.Router, _userManager manager.UserManager) {
+func UserRouter(router *mux.Router, _userManager manager.User) {
 	validate = validator.New()
 	userManager = _userManager
```

Comment:
Why we need to add `_` as `userManager` prefix and why we re-declare it as `userManager` variable?

Reply:
sorry that's my habit, alywas use perefix `_` in function params, if name same with property/variable class.  will change it without `_`.

Reply:
updated